apps/artemis-web/components/hijack-info/hijack-info.tsx

```diff
@@ -333,6 +400,28 @@ class HijackInfoComponent extends Component<any, any> {
                   </div>
                 </div>
               </div>
+
+              <div className="row">
+                <div className="col-lg-12">
+                  <div className="card" style={{ marginTop: '12px', visibility: this.state.gripState ? "visible" : "hidden" }}>
+                    <div className="card-header">
+                      CAIDA GRIP{' '}
+                    </div>
+                    <div className="card-body">
+                      <button
+                        style={{ marginRight: '5px', marginTop: '5px', float: 'left' }}
+                        id="edit_comment"
+                        type="button"
+                        className={`btn btn-primary
+                          } btn-lg`}
+                        onClick={() => window.open(`https://grip-dev.caida.org/events/${type}/${event_id}`, "_blank")}
```

Comment:
this opens up a separate tab, correct? Also, if no GRIP event could be retrieved, what is the visual behavior of this in the hijack info page? (please add screenshot)

Reply:
Also, what happens if multiple GRIP events are retrieved? (several IDs I mean). Could you double-check the API call? We can select one of them if needed.

Reply:
interesting thought! I just fetch the first one.

We could add multiple buttons if they are more? Or a dropdown menu?

Reply:
drop-down menu would be fine